lighthouse-core/config/config.js

```diff
@@ -437,9 +437,15 @@ class Config {
       assertValidPluginName(configJSON, pluginName);
 
       const pluginPath = resolveModule(pluginName, configDir, 'plugin');
+
       const rawPluginJson = require(pluginPath);
       const pluginJson = ConfigPlugin.parsePlugin(rawPluginJson, pluginName);
 
+      if (rawPluginJson['localePath']) {
+        const pluginParentPath = 
+          pluginPath.replace(new RegExp(`/${pluginName}/.*`,'g'), '');
+        i18n.mergeLocales(rawPluginJson['localePath'], pluginParentPath);
```

Comment:
at this point, we know what the locale will be (`configJSON.settings.locale`). could we just merge a single locale (rename to `mergeLocale`) instead of everything?